crimsobot/cogs/utilities.py

```diff
@@ -41,6 +41,8 @@ async def poll(self, ctx: commands.Context, *, poll_input: str) -> None:
         if len(question) == 0:
             question = 'QUICK POLL!'
 
+        attachment = None  # prevent unbound local var error
```

Comment:
Why is attachment = None added again?

Reply:
oh right i didn't notice that... :thonk: